### DISCUSSION TITLE
http only cookie 를 사용하지 못하는 이유

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#310] [03.Technical]
# http only cookie 를 사용하지 못하는 이유
![image](https://user-images.githubusercontent.com/74234333/193547136-42eb27fc-42be-45d6-9dfe-51452afc37f4.png)

**http only cookie**의 경우 서버에서만 접근 가능합니다.
하지만 우리 로직의 경우, 필연적으로 쿠키에 접근해야하므로 (Backend 의 api가 그렇게 짜져 있어서) 
http only cookie를 활용할 수 없습니다 ㅠㅠ

**Comment by gitul0515** [comment]
ㅠㅠ

**Comment by Hyevvy** [comment]
ㅠㅠ
